include/ppx/application.h

```diff
@@ -394,6 +394,21 @@ class Application
     {
         return GetSwapchain(mUISwapchainIndex);
     }
+
+    ppx::float4x4 GetViewProjectionMatrix(uint32_t viewIndex, float nearZ, float farZ)
+    {
+        ppx::float4x4 viewProjectionMatrix(1);
+        viewProjectionMatrix = mXrComponent.GetViewProjectionMatrix(viewIndex, nearZ, farZ);
+        return viewProjectionMatrix;
+    }
+
+    uint32_t GetCurrentViewIndex() const
+    {
+        uint32_t current_view_index = 0;
+        current_view_index          = (IsXrEnabled() ? mXrComponent.GetCurrentViewIndex() : 0);
+
+        return current_view_index;
+    }
```

Comment:
Instead of adding these, can we incorporate them into and go through XrCamera here: https://github.com/google/bigwheels/blob/7d2f93755c57f3dc0c24f805a7552af678017ab0/src/ppx/xr_component.cpp#L871
Otherwise I think we are reverting some changes made here https://github.com/google/bigwheels/pull/391

Reply:
Removed these functions and just utilized the xrComponent.